src/Core/Service/ServiceException.php

```diff
@@ -133,6 +136,19 @@ public static function missingAppVersionInfo(): self
         );
     }
 
+    /**
+     * @param non-empty-array<int, string> $missingFields
+     */
+    public static function missingAppVersionInformation(array $missingFields): self
```

Comment:
```suggestion
    public static function missingAppVersionInformation(string ...$missingFields): self
```

if this is a list of strings, we should make it type safe :blush: